study.md

```diff
@@ -135,15 +135,17 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a breif description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP is a system that tells what type of messages are passed between clients
+and servers
 ```
```

Comment:
It doesn't necessarily tell what is passed but _is_ the set of rules those transfers adhere to.